MyOnlineStore/ruleset.xml

```diff
@@ -13,6 +13,7 @@
         <exclude name="Symfony.Commenting.FunctionComment.Missing"/>
         <exclude name="Symfony.Commenting.FunctionComment.MissingParamTag"/>
         <exclude name="Symfony.Commenting.FunctionComment.MissingReturn"/>
+        <exclude name="Symfony.Commenting.FunctionComment.ParamNameNoMatch"/>
```

Comment:
this will also allow for non related (faulty) params to be listed in docblocks?

Reply:
@frankverhoeven ?

Reply:
Not sure, will have to look into it.

Reply:
Yes, it does allow for invalid params. Personally I don't think of it as a big problem, because we basically never add `@param` annotations unless strict typing is not possible, or the types of an array have to be defined.
I could look into writing a sniff for this next week though, don't think it is a lot of work.

Reply:
yes that would help of course! I personally always check docblocks but it is easy to miss a fault entry and automated testing is always better! :+1: